docs/how-to/rocm-for-ai/train-a-model.rst

```diff
@@ -0,0 +1,138 @@
+.. meta::
+   :description: How to use ROCm for AI
+   :keywords: ROCm, AI, LLM, train, fine-tune, FSDP, DeepSpeed, LLaMA, tutorial
+
+****************
+Training a model
+****************
+
+The following is a brief overview of popular component paths per AI development use-case, such as training, LLMs,
+and inferencing.
+
+Accelerating model training
```

Comment:
Do we need more content on the basics of training a model with ROCm before jumping straight into optimization with distributed training?